### PR TITLE
New article: Santa's Helpers Embrace the Shell

### PR DESCRIPTION
**Pull request by s1037989**
Let me know what I'm doing wrong with my first Advent PR!

### CODE REVIEW

**Comment by oalders** [comment]
@thibaultduponchelle maybe don't merge this after your review as I'm realizing it would generally be good to have 2 reviewers per article.

**Comment by thibaultduponchelle** [comment]
@oalders Ok let’s use this policy 👍🏼

**Comment by thibaultduponchelle** [comment]
I'm going to check code snippets execution and read a second time the article but we really need a native English speaker to review it.

Any volunteer?

**Comment by openstrike** [comment]
I (native English speaker) can review it for the prose if you want?

**Comment by thibaultduponchelle** [comment]
> I (native English speaker) can review it for the prose if you want?

Oh yes please do 🙏🏼

File: 2022/incoming/santas-helper-embrace-the-shell.pod
Comment (openstrike) [inline 2022/incoming/santas-helper-embrace-the-shell.pod:9]: **tasks** would be better than tasking here.

File: 2022/incoming/santas-helper-embrace-the-shell.pod
Comment (openstrike) [inline 2022/incoming/santas-helper-embrace-the-shell.pod:34]: "to get at any specific data in their JSON" - I guess this means any specific data set in which case perhaps just explicitly say "data set" here.

File: 2022/incoming/santas-helper-embrace-the-shell.pod
Comment (openstrike) [inline 2022/incoming/santas-helper-embrace-the-shell.pod:53]: "They understand that there may be" is missing "that"

**Review by openstrike — COMMENTED** [review]
This article reads really well. I have made a few very minor recommendations but would be happy enough to merge as-is if others want to proceed.

**Comment by thibaultduponchelle** [comment]
I'm good with this article. Tested the code snippets 👍🏼 

I have 2 remarks not related to the content of the article but `jp` tool:
1. I think it should be uploaded to CPAN
2. You could add a `cpanfile` (e.g. for installing `Data::Walk::More`)

I going to merge it now.

Thank you a lot @s1037989 let's time to start writing another article? 🤣

**Review by thibaultduponchelle — APPROVED** [review]

**Comment by thibaultduponchelle** [comment]
@openstrike Can you please "approve" this PR so I can merge it (mandatory 2 approvals)

**Review by openstrike — APPROVED** [review]
Approved.

**Pull request merged**
